docs/concepts/json_schema.md

```diff
@@ -1029,6 +1076,94 @@ print(json.dumps(TypeAdapter(Person).json_schema(), indent=2))
 ```
 
 
+### Using `field_title_generator`
+
+The `field_title_generator` parameter can be used to dynamically generate the title for a field based on its name.
+This is similar to the field level `field_title_generator`, but the `ConfigDict` option will be applied to all fields of the class.
+
+See the following example:
+
+```py
+import json
+
+from pydantic import BaseModel, ConfigDict
+from pydantic.alias_generators import to_pascal
+
+
+class Person(BaseModel):
+    model_config = ConfigDict(field_title_generator=to_pascal)
```

Comment:
Maybe we could use a different generator here - `to_pascal` results in the same names as the default capitalization for `name` and `age`

Reply:
Could either go with `upper` or something like your example in the class title case below :)